src/plugins/polar/README.md

```diff
@@ -0,0 +1,96 @@
+Polar Coordinate Positioning Plugin
+===================================
+
+Preface
+-------
+This plugin includes and enhances functionality from the rel plugin.
+Please only use one of them.
+
+Relative Positioning (Cartesian)
+--------------------------------
+The plugin uses the following attributes for a step:
+	data-rel-ref
+	data-rel-x
+	data-rel-y
+	data-rel-z
+
+If the attribute "data-rel-ref" is not given, relative coordinates will be measured
+from the position of the previous frame. If added, the step with the given id determines
+the origin. This allows to group steps to a topic without the risk of scattering them when 
+one of them is removed.
+In addition to plain numbers, which are pixel values, it is also possible to
+define relative positions as a multiple of normal step height and width, using
+a unit of "h" or "w", respectively, appended to the number.
+
+Example:
+	<div id="relstep" class="step" 
+		data-rel-ref="main_step" data-rel-x="1w" data-rel-y="0.5h" data-rel-z="0">
+			
+Relative coordinates not given are inherited from the previous step, so make sure
+to set them to zero if that behaviour is not desired.
+Absolute values always override relative values!
+
+Polar Positioning
+-----------------
+
+To place steps e.g. in a circle around a central thought, polar positioning is introduced here.
+It uses the following attributes for a step:
+	data-rel-ref
+	data-rho
+	data-phi
+	data-rel-rho
+	data-rel-phi
+
+Rho is the radius (the distance from the origin) and phi the angle in degree, counting from the 
+horizontal (positive x, "03:00hrs") counterclockwise.
+In addition to plain numbers, which are pixel values, it is also possible to define the radius 
+as a multiple of the normal step diagonal, using a unit of "d" appended to the number.
+
+Relative polar coordinates are not added up, they just count from the given origin. For absolute
+coordinates that is (0,0,0), for relative ones the origin of the last step or the one pointed to 
+by the "data-rel-ref" attribute. 
+Alternatively, a new relative origin can be set by giving absolute cartesian coordinates.
+
```

Comment:
What if no `data-rel-ref` attribute is given, but `data-rel-rho` and `data-rel-phi` attributes are present?
Doesn't that mean, that the coordinates are meant relative to the previous step? In such a way the would kind of "add up", if multiple steps are defined that way, at least I would expect that.

Reply:
Sorry, Must be `data-pos-ref`. This is only used for 'absolute' coordinates. 
Relative coordinates are always from the last step and are inherited and add up until some other positioning is defined. This is the same as I understand the original rel plugin is supposed to work.

And yes, it does the same as the rel plugin, as all positioning must be done for every step at the same time - otherwise no reference would work if a later plugin relocates the referred step. Therefore it's either the rel plugin or this one.